gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>Can you add a link to the video? (If it exists now)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Video isn't ready, and so I will remove the line for now. Thanks for catching that!</p>